config/webpack.prod.js

```diff
@@ -29,6 +29,7 @@ const OfflinePlugin = require('offline-plugin');
  * Webpack Constants
  */
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
+const FABRIC8_FORGE_API_URL = process.env.FABRIC8_FORGE_API_URL || 'https://forge.api.prod-preview.openshift.io';
```

Comment:
You need to change 

```typescript
   readonly DEFAULT_API_PREFIXES = new Map<string, string>([
      ['wit', 'api'],
      ['recommender', 'recommender'],
     ['sso', 'sso'],
     ['forge', 'forge']
    ]);
```

to

```typescript
   readonly DEFAULT_API_PREFIXES = new Map<string, string>([
     ['wit', 'api'],
     ['recommender', 'recommender'],
     ['sso', 'sso'],
     ['forge', 'forge.api']
    ]);
```

And then you can remove the override from the prod webpack. The api-locator will automatically configure the right URL for you.